app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -517,19 +528,9 @@ private fun RunHikeBottomSheet(
               modifier = Modifier.testTag(RunHikeScreen.TEST_TAG_HIKE_NAME))
 
           // Elevation graph and the progress details below the graph
+          val hikeColor = Color(hike.color)
+          ElevationGraphOnPercentage(completionPercentage, hike, hikeColor)
```

Comment:
Question: Why is the new elevation graph not in the column?

Reply:
Big error my part. Sorry. Fixed